dts/arc/tenstorrent/tt_blackhole_smc.dtsi

```diff
@@ -95,6 +95,23 @@
 		ngpios = <16>;
 	};
 
+	ref_clk: ref_clk {
+		compatible = "fixed-clock";
+
+		clock-frequency = <1000000000>; /* 1 GHz */
+		#clock-cells = <0>;
+
+		status = "okay";
+	};
+
+	pll: pll@80020100 {
```

Comment:
I think technically, there should be several instances of this. Can it be converted in a follow-up PR?

Reply:
I thought it would be better to have one PLL device since the init function depends on initializing all the PLLs together. I wasn't sure if I could separate the initialization for each PLL

Reply:
In the `init()` function, if something has to be done on the first call to all of the PLLs, or some common registers, then the driver could probably have a static `bool common_init_complete;` that gets set after a common init function is done once. Something like

```cpp
static void clock_control_tt_bh_init_common(void)
{
  static bool complete;
  
  if (complete) {
    return;
  }
  
  /* common init here */
}
```